miniapp/src/test/java/com/rakuten/tech/mobile/miniapp/api/RetrofitRequestExecutorSpec.kt

```diff
@@ -201,6 +198,48 @@ open class RetrofitRequestExecutorErrorSpec : RetrofitRequestExecutorSpec() {
         executor.executeRequest(request)
     }
 
+    @Test(expected = MiniAppNetException::class)
+    fun `should throw exception when there is network error`() = runBlockingTest {
+        val executor = spyRetrofitExecutor()
+        val request: Call<String> = object : Call<String> {
```

Comment:
Couldn't you do this mocking instead?

```
val request: Call<String> = mock()
When calling request.execute() itThrows SocketTimeoutException()
```

Reply:
If you do mocking in function to throw exception, you must declare the throwing exception in method. For example, 
```
@Throws(SocketTimeoutException::class)
suspend fun <T> executeRequest(call: Call<T>)
```
but I guess we don't want to change this in our source code.